tests/test_blob_functions.py

```diff
@@ -1,6 +1,6 @@
 import time
 
-from azure.worker import testutils
+from azure._functions.worker import testutils
```

Comment:
_functions? is this intentional instead of azure.functions.worker?

Reply:
From @elprans (https://github.com/Azure/azure-functions-python-worker/issues/140) - 
> We cannot move the worker under azure.functions as we are separating the packages, and azure.functions is not a pure namespace. I suggest renaming the worker to azure._functions.worker instead.

Reply:
would azure.functionsWorker work?

Reply:
Also, can you please clarify what is pure namespace in python? So we have azure.functions and azure.functions.worker does it mean they are related?

Reply:
A pure namespace is a package that only contains other packages (no classes).  Since `azure.functions` contains classes, it's not a pure namespace.

Reply:
Changed to `azure.functions_worker`